src/include/stir/SeparableGaussianArrayFilter.h

```diff
@@ -0,0 +1,68 @@
+//
+// $Id: 
+//
+/*!
+
+  \file
+  \ingroup ImageProcessor
+  \brief Declaration of class stir::SeparableGaussianArrayFilter
+
+  \author Sanida Mustafovic
+  \author Kris Thielemans
+
+   $Date: 
+   $Revision: 
+*/
+/*
+    Copyright (C) 2000- 2002, IRSL
+    See STIR/LICENSE.txt for details
+*/
+
+#ifndef __stir_SeparableGaussianArrayFilter_H__
+#define __stir_SeparableGaussianArrayFilter_H__
+
+#include "stir/DiscretisedDensity.h"
+#include "stir/SeparableArrayFunctionObject.h"
+#include "stir/Array.h"
+#include "stir/BasicCoordinate.h"
+
+#include <vector>
+
+START_NAMESPACE_STIR
+
+
+template <int num_dimensions, typename elemT>
+class SeparableGaussianArrayFilter:
+      public SeparableArrayFunctionObject <num_dimensions,elemT> 
+{
+public:  
+
+  //! Default constructor
+  SeparableGaussianArrayFilter();  
+  
+
+  SeparableGaussianArrayFilter(const BasicCoordinate< num_dimensions,float>&  fwhm,
+                               const BasicCoordinate< num_dimensions,int>&  max_kernel_sizes,
+                               bool normalise = false);
```

Comment:
default has to be same as in set_defaults, so if we change it there, we should here